Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -83,8 +85,12 @@ bool ProcessThrottler::addActivity(Activity& activity)
 void ProcessThrottler::removeActivity(Activity& activity)
 {
     ASSERT(isMainRunLoop());
-    if (activity.isForeground())
+    if (activity.isForeground()) {
         m_foregroundActivities.remove(&activity);
+        // If this was the last foreground activity keep holding the foreground assertion for a while
```

Comment:
Comments need to end with a period in WebKit, per coding style. Also, I think it would read better with a comma after "If this was the last foreground activity".